feature/settings/src/main/java/com/google/jetpackcamera/settings/ui/SettingsComponents.kt

```diff
@@ -389,6 +391,64 @@ fun StreamConfigSetting(
     )
 }
 
+@Composable
+fun LowLightBoostPrioritySetting(
+    lowLightBoostPriorityUiState: LowLightBoostPriorityUiState,
+    setLowLightBoostPriority: (LowLightBoostPriority) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    BasicPopupSetting(
+        modifier = modifier.testTag(BTN_OPEN_DIALOG_SETTING_LOW_LIGHT_BOOST_PRIORITY_TAG),
+        title = stringResource(R.string.low_light_boost_priority_title),
+        leadingIcon = null,
+        enabled = true,
+        description =
+        if (lowLightBoostPriorityUiState is LowLightBoostPriorityUiState.Enabled) {
+            when (lowLightBoostPriorityUiState.currentLowLightBoostPriority) {
+                LowLightBoostPriority.PRIORITIZE_AE_MODE -> stringResource(
+                    id = R.string.low_light_boost_priority_description_ae_mode
+                )
+
+                LowLightBoostPriority.PRIORITIZE_GOOGLE_PLAY_SERVICES -> stringResource(
+                    id = R.string.low_light_boost_priority_description_google_play_services
+                )
+            }
+        } else {
+            TODO("low light boost priority currently has no disabled criteria")
```

Comment:
As mentioned in another comment, we should add different Illuminants for the 2 different flavors of LLB. If only 1 or neither of the LLB illuminants are available, this setting should be disabled. When only 1 is available, there is no reason to present the option to the user, which is why it will also be disabled in that case.